src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableSingleOpEntity.java

```diff
@@ -81,23 +107,37 @@ public Object decode(ByteBuf buf) {
         // 0. decode header
         super.decode(buf);
 
-        int len = (int) Serialization.decodeVi64(buf);
-        for (int i = 0; i < len; i++) {
-            String column = Serialization.decodeVString(buf);
-            rowKeyNames.add(column);
+        rowKeyBitLen = Serialization.decodeVi64(buf);
+        rowKeyBitMap = new byte[(int) Math.ceil(rowKeyBitLen / 8.0)];
```

Comment:
could you abstract it into a function and add comment?

Reply:
Fixed, has set into `isSamePropertiesColumnNamesLen()`